components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/FetchBuildValidationDataCommand.java

```diff
@@ -206,28 +223,27 @@ private void logStartFetchingBuildScan(int index) {
         }
     }
 
-    private String fetchingMessageFor(int index) {
-        if (buildScanUrls.size() <= 10) {
-            return String.format("Fetching build scan data for the %s build", buildScanIndexToOrdinal(index));
+    private String fetchingMessageFor(int runNum) {
+        if (runNumsAndBuildScanUrls.size() <= 10) {
```

Comment:
I would pass in `runNumsAndBuildScanUrls` to this method. IMO, the params passed originally get processed in call() and should not be accessed as instance fields anymore from then on.

Reply:
✅ Eliminating the size check eliminates the need to access runNumsAndBuildScanUrls at all.  In fact, eliminating that size check reduced this method to a single line, which only called in one place, so I inlined it.